web/src/people.ts

```diff
@@ -459,6 +459,18 @@ export function get_recipients(user_ids_string: string): string {
     return names.join(", ");
 }
 
+export function get_recipients_array(user_ids_string: string): string[] {
```

Comment:
Maybe a name like `get_direct_message_recipient_names_list` would be appropriately explicit?

Reply:
Apparently, I decided not to use this function. removed.